lambdas/replay-cimit-vcs/dynamo-scripts/scan.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+Help()
+{
+   # Display Help
+   echo "Syntax: scriptTemplate [-e|p]"
+   echo "options:"
+   echo "e     Specifies the environment label for the script output"
+   echo "p     Specifies the AWS profile to use with the script"
+   echo
+}
+
+# Set variables
+Env="build"
+Profile="None"
+
+# Script options
+while getopts ":he:p:" option; do
+   case $option in
+      h) # display Help
+         Help
+         exit;;
+      e) # Enter an environment
+         Env=$OPTARG;;
+      p) # Enter an AWS profile
+         Profile=$OPTARG;;
+     \?) # Invalid option
+         echo "Error: Invalid option"
+         exit;;
+   esac
+done
+
+echo Performing scan for $Profile in $Env
+
+# Scan script
+aws dynamodb scan \
+--table-name user-issued-credentials-v2-$Env \
+--filter-expression "attribute_exists(dateCreated) AND credentialIssuer = :vc" \
```

Comment:
Can we filter on date here as well? Presumably as written, this will dump the details of every single VC in the store, which will be massive!

Even if it doesn't change the cost, filtering dynamo-side will save a load of unnecessary data transfer.

Reply:
I'll have a go at adjusting as you've suggested here and run against build again.